Makefile

```diff
@@ -2,11 +2,17 @@ GITHUB=$(shell opam config var github:installed)
 APP=Datakit.app
 EXE=Datakit
 
+COMMIT_ID=$(shell git rev-parse HEAD)
```

Comment:
The release archives should not call `make COMMIT` (hopefully). As `make` is lazy, that should be fine.